crates/hc_sandbox/CHANGELOG.md

```diff
@@ -4,6 +4,8 @@ The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/).
 
 ## \[Unreleased\]
 
+- **BREAKING CHANGE** - `hc sandbox` updated to use new (0.y.z) lair api. Any old sandboxes will no longer function. To manually update them, run `lair-keystore init` in the `[sandbox]/keystore` directory and update the keystore section of the holochain conductor config. [\#1515](https://github.com/holochain/holochain/pull/1515)
```

Comment:
nice, this is definitely helpful!

Reply:
now that i think about this, I wonder if we should include this... No keys will be transferred by this, so it's only good for sandboxs dirs that are otherwise uninitialized...

Reply:
updated to:

- **BREAKING CHANGE** - `hc sandbox` updated to use new (0.y.z) lair api. Any old sandboxes will no longer function. It is recommended to create new sandboxes, as there is not a straight forward migration path. To migrate: [dump the old keys](https://github.com/holochain/lair/blob/v0.0.11/crates/lair_keystore/src/bin/lair-keystore/main.rs#L38) -> [write a utility to re-encode them](https://github.com/holochain/lair/tree/hc_seed_bundle-v0.1.2/crates/hc_seed_bundle) -> [then import them to the new lair](https://github.com/holochain/lair/tree/lair_keystore-v0.2.0/crates/lair_keystore#lair-keystore-import-seed---help) -- [\#1515](https://github.com/holochain/holochain/pull/1515)

Reply:
great, it's better to set realistic expectations as early as possible